beakerx/beakerx/tabledisplay/tableitems.py

```diff
@@ -43,9 +44,9 @@ class ColumnType(Enum):
 class DateType:
     type = "Date"
 
-    def __init__(self, value):
-        self.timestamp = parse(str(value)).timestamp() * 1000
-
+    def __init__(self, value, tz=None):
+        self.timestamp = parse(str(value)).replace(tzinfo=timezone.utc).timestamp() * 1000
```

Comment:
@LeeTZ parsed date was treated as local time so timestamp was shifted.  Maybe it can be done better. Please take a look.